tests/docker/docker-compose.yml

```diff
@@ -1,4 +1,5 @@
 version: '3'
```

Comment:
I think this line can be removed from all docker-compose.yml files.

Reply:
done